ui/lib/server/widget/message.dart

```diff
@@ -14,7 +14,7 @@ class Message extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           Row(children: [
-            Text(message.user),
+            Text(message.user + ' '),
```

Comment:
Instead of concattenating string (whose size is dependent on fontSize) insert a ```SizedBox``` with appropriate width after the ```Text(message.user)```.